raiden/network/proxies/token_network.py

```diff
@@ -985,7 +984,6 @@ def close(
             )
 
             if gas_limit:
-                error_prefix = 'closeChannel call failed'
```

Comment:
These `error_prefix` and `error_type` were just a confusing control flow with very little added value. I streamlined the code a bit, added proper checks, and improved the error messages.